src/core.js

```diff
@@ -212,7 +212,10 @@ jQuery.extend( {
 	noop: function() {},
 
 	isFunction: function( obj ) {
-		return jQuery.type( obj ) === "function";
+
+		// Support: Chrome <=57, Firefox <=52
+		// Don't classify callable <object>s as functions
+		return typeof obj === "function" && typeof obj.nodeType !== "number";
```

Comment:
If `<object>` is referenced to `typeof` result for objects, how it could bypass out initial `typeof obj === "function"`?

Then `function` type is a subtype of `object` so why we can't classify function object as function?
Most of all, why we check presence of `nodeType` property to see if `object` is callable and why do we care?

"elements" part provides some clarification, but only if you know that this is the reference to the DOM elements. And even so why "element"? Why not "node"? If the logic is meant only for elements indeed then how come it cannot reach this part for other nodes? I think it also deserves some explanation as well as some examples of such an input.

I think I know answers to these questions, but after some months I might forget, you might forget, and others who going to read this might not know of course